### PR TITLE
Feature: Added Files icon to the process in Windows

### PR DESCRIPTION
**Pull request by IviriusMain**
**Resolved / Related Issues**

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- Closes #[16320](https://github.com/files-community/Files/issues/16320)

**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Open Files
2. Open Task Manager
3. The process now has an icon. NOTE: This icon is located at `\src\Files.App\Logo.ico` and it should always be replaced with the icon of the version launched to the public or as an insider build.

Why is this important?

In order for users to find Files more easily and be more comfortable with the app, the task should have an appropriate icon.

### CODE REVIEW

File: src/Files.App/Logo.ico
Comment (0x5bfa) [inline src/Files.App/Logo.ico:1]: Please use the ico in [`src/Files.App (Package)/Assets/AppTiles/Dev/Logo.ico
`](https://github.com/files-community/Files/blob/main/src/Files.App%20(Package)/Assets/AppTiles/Dev/Logo.ico)
Thiguh you hardcode Dev version, the CI replaces with the appropriate branch version.

**Review by 0x5bfa — COMMENTED** [review]
Please send a screenshot what the app looked/looks like in Task Manager.

**Comment by IviriusMain** [comment]
Expanded view for better reference (same thing: above is the latest Insider build, below is the build in this pull request)

File: src/Files.App/Logo.ico
Comment (IviriusMain) [reply src/Files.App/Logo.ico:1]: I did use exactly that, so all good?

Comment (0x5bfa) [reply src/Files.App/Logo.ico:1]: No use that instead of copying.

Comment (IviriusMain) [reply src/Files.App/Logo.ico:1]: Done, I pushed the fix.

**Review by 0x5bfa — APPROVED** [review]
Looks fine.
This is what I was doing when I tried to introduce Directory.Build.props

**Review by yaira2 — APPROVED** [review]

**Comment by yaira2** [comment]
@IviriusMain congrats on your first pull request! Thank you!

**Pull request merged**